spec/features/carts/show_spec.rb

```diff
@@ -16,18 +16,40 @@
 # I also see a grand total of what everything in my cart will cost
 RSpec.describe 'As a visitor' do
   context 'As a visitor' do
-    before :each do
-      @user = User.create!(email: "test@test.com", password_digest: "t3s7", role: 1, active: true, name: "Testy McTesterson", address: "123 Test St", city: "Testville", state: "Test", zip: "01234")
-
-      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
-    end
 
     it 'When my cart is empty and I visit my cart, It says my cart is empty' do
       visit carts_path
 
       expect(page).to have_content('Your cart is currently empty.')
       expect(page).to_not have_content('Empty your Cart')
     end
+
+    it 'shows all items in my cart and allows user to clear cart' do
+      merchant = create(:user, name: "Merchant", role: 1)
+      item_1 = create(:item, user: merchant)
+
+      visit items_path
+
+      within "#item-#{item_1.id}" do
+        click_button "Add To Cart"
```

Comment:
@zachlp2016 Vince just changed these to links, do you mind changing this click_button to click_link?

Reply:
Awesome just changed it thanks Brennan!

Reply:
Thanks!